src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1750,6 +1750,14 @@ public static boolean isXnp(final String reference, final String alternate) {
         return ((refComparable.length() == altComperable.length()) && (!refComparable.equals(altComperable)));
     }
 
+    /**
+     * @param vc {@link VariantContext to test}
+     * @return true if the only alternate allele for this VariantContext is a spanning deletion, otherwise false.
+     */
+    public static boolean isSpanningDeletionOnly(final VariantContext vc){
+        return vc.getAlternateAlleles().size() == 1 && vc.getAlternateAllele(0).basesMatch(Allele.SPAN_DEL);
```

Comment:
This should probably call `GATKVCFConstants.isSpanningDeletion`.  It's not done uniformly but it should probably be made to be uniform.

Reply:
Done.